### PR TITLE
Map changes and Remove fugitive from traitor gamemode

### LINKED ISSUE #5961: invisible bullets and no fireing effect
### Environment
- Game Version: up to date potatostation (at least i think it was potatostaion but im unsure, if you can check which server had players around 9 or so GMT) as of 07/02/2021 (english date system)
- OS Version: windows 10 up to date as of previously mentioned date

### Detailed description of bug

i was trying to break into cargo to steal some glass, i asked a trator for a gun and he gave me a lazer rifle, a riot shotgun and a pistol, the lazer rifle didnt work but the pistol and shotgun did, they managed to break some glass and i got the text thing saying they were shooting but there was no visual effect, i dont know about audio as the game was muted at the time.

### Steps to reproduce

1. get a trator to give you a pistol in fallstation
2. go to the enterance to cargo
3. start shooting at the glass

### LINKED ISSUE #6008: Exit button in menu is not working
### Environment
- Game Version: latest staging
- OS Version: windows 10

### Detailed description of bug

exit button in the in-game menu does not work when clicked.

### Steps to reproduce

1. join a round
2. hit escape to open in game menu
3. click the exit button in the corner of the window
4. the menu does not close.

### LINKED ISSUE #6009: Players can still randomly spawn as Fugitives at round start.
### Environment
- Game Version: latest staging ( build 21021202)

### Detailed description of bug

Fugitives have recently been removed from the player antag preferences and moved to be a ghost role event. However, It is still possible to randomly spawn as a Fugitive the first time a player joins a round.

### PR DESCRIPTION
**Pull request by ThatDan123**
CL: Removed fugitive from traitor gamemode

CL: Adds some emitters and other stuff to box, square and outpost, not full PA set up

CL: Fixed escape and close button not working on menu

CL: Tesla now moves through walls and does correct damage

Fixes some miscellaneous NREs

fixes #6009 

fixes #6008

Hopefully fixes #5961, haven't build tested but it seems like i fixed the issue found using breakpoints

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/TeslaEnergyBall.cs [hunk 1]
```diff
@@ -335,7 +335,7 @@ private void OnPulse(ElectricalArc arc)
 			}
 			else if (arc.Settings.endObject.TryGetComponent<Integrity>(out var integrity) && integrity != null && integrity.Resistances.LightningDamageProof == false)
 			{
-				integrity.ApplyDamage(damage * ((int)currentStage + 1), AttackType.Magic, DamageType.Burn, explodeOnDestroy: true);
+				integrity.ApplyDamage(damage * ((int)currentStage + 1), AttackType.Magic, DamageType.Burn, true, explodeOnDestroy: true);
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/TeslaEnergyBall.cs:338]: tesla does magic damage?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Objects/TeslaEnergyBall.cs:338]: i copied and changed around nooneys lightning spell, not sure what damage it does in tg

Comment (corp-0) [reply UnityProject/Assets/Scripts/Objects/TeslaEnergyBall.cs:338]: ya, good enough for now. We can always change it if we need it.

**Comment by ThatDan123** [comment]
I removed the AimApply check, so for now it will always send the interaction and not null

**Comment by corp-0** [comment]
ok ok, looks good. Making a note here about how we need to change interactions with Bod but for now and since we're in an emergency, I will merge.

**Pull request merged**